src/Squirrel/ReleasePackage.cs

```diff
@@ -160,34 +160,42 @@ public string CreateReleasePackage(string outputFile, string packagesRootDir = n
         // We must do this, or PathTooLongException may be thrown for some unicode entry names.
         public static void ExtractZipDecoded(string zipFilePath, string outFolder, string directoryFilter = null)
         {
-            var zf = new ZipFile(zipFilePath);
+            Utility.Retry(() =>
```

Comment:
This will probably work, but it means redoing the whole (100m or so?) extract on every failure. I would have been more inclined to put the Retry calls just (or also) around the directory creation and the individual file extracts.
Will the file extract operations succeed if (on a retry) the file already exists?